client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,18 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Sorry, themes upload is not yet available for multi-site installations.' ) }
+				line={ this.props.translate( 'Open your site:' ) }
```

Comment:
I didn't see this is the screenshot. Usually the `title` is short, and the `line` gives more explanation. This is a pretty good one I think:

![screen shot 2017-02-22 at 10 15 10 am](https://cloud.githubusercontent.com/assets/618551/23220508/db68779e-f8e7-11e6-8757-9834029c27ba.png)